zt_backend/models/components/list.py

```diff
@@ -0,0 +1,128 @@
+
+
+class ListComponent(ZTComponent):
+    """List the Primary components holding the items in the scorllable format"""
+
+    component: str = Field("v-list", description="Vue component name for List")
+    childComponents: List[str] = Field(
+        [], description="List of child component ids to be placed within the List"
+    )
+    color: str = Field(None, description="Background color of the List")
+    elevation: int = Field(
+        None,
+        ge=0,
+        le=24,
+        description="Elevation level of the List. Must be between 0 and 24",
+    )
+    density: str = Field(
+        None,
+        enum=["default", "comfortable", "compact"],
+        description="Density of the List",
+    )
+    width: Union[int, str] = Field("100%", description="Width of the List")
+    height: Union[int, str] = Field("100%", description="Height of the List")
+
+
+class ListItem(ZTComponent):
+    """List Item helps define properties of individual items in the list"""
+
+    component: str = Field(
+        "v-list-item", description="Vue component name for List Item"
+    )
+    title: str = Field("", description="item title")
+    color: str = Field(None, description="Background color of the List item")
+    elevation: int = Field(
+        None,
+        ge=0,
+        le=24,
+        description="Elevation level of the List item. Must be between 0 and 24",
+    )
+    density: str = Field(
+        "default",
+        enum=["default", "comfortable", "compact"],
+        description="Density of the List Item: default | comfortable | compact",
+    )
+    width: Union[int, str] = Field("100%", description="Width of the List item")
+    height: Union[int, str] = Field("100%", description="Height of the List item")
+    value: str = Field("", description="value for List Item")
+    disabled: bool = Field(False, description="removes ability to click List Item")
+    childComponents: List[str] = Field(
+        [],
+        description="List of child component ids to be placed within the ListItem. List title, subtitle come as the child components of the list item",
+    )
+
+
+class ListGroup(ZTComponent):
+    """List Group is used to group multiple list items in one category"""
+
+    component: str = Field(
+        "v-list-group", description="Vue component name for List group"
+    )
+    title: str = Field("", description="Title of the List Group")
+    color: str = Field(None, description="Background color of the List Group")
+    density: str = Field(
+        "default",
+        enum=["default", "comfortable", "compact"],
+        description="Density of the component",
+    )
+    width: Union[int, str] = Field("100%", description="Width of the List")
+    height: Union[int, str] = Field("100%", description="Height of the List")
+    value: str = Field("", description="Expands / Collapse the list-group.")
+    disabled: bool = Field(False, description="removes ability to click List Item")
+    collapse_icon: str = Field(
```

Comment:
Do not need to include icons.  Default is fine

Reply:
@metc1999 @HarshitaDPoojary

Reply:
Removed in [commit 644eb127e48c34edf17c1178188def6aa83e650d](https://github.com/Zero-True/zero-true/pull/277/commits/644eb127e48c34edf17c1178188def6aa83e650d#diff-8ae68a9877ed8ca0c9e1390b387042a689df0f149f2ac5e600d1280b18f00ac7L58)